### DISCUSSION TITLE
Question about lympha flow and operation about PBPK modeling for large molecule

### DISCUSSION THREAD

**Discussion by wangwei1619** [Open-Systems-Pharmacology/Forum#795] [Questions & Problems] [answered]
Hi, all 

Recently I have read the paper "Niederalt, C., Kuepfer, L., Solodenko, J., Eissing, T., Siegmund, H. U., Block, M., ... & Lippert, J. (2018). A generic whole body physiologically based pharmacokinetic model for therapeutic proteins in PK-Sim. Journal of pharmacokinetics and pharmacodynamics, 45(2), 235-257." 

I find only lymph flow in text and no term such as "lymph volume" is referred. Is there no compartment of lymph with volume in PBPK model for marcomolecule? Is this to say the distribution of such agent in lymph tube is trival?

Is there any step-by-step tutorial about  PBPK modeling for large molecule when target-mediated binding and clearance is concerned?

Thanks for any reply.

**Comment by Christoph27** [comment, marked as answer]
Dear Wang.

you are right, there is no compartment representing lymphatic vessels or lymph nodes. This does not mean that concentrations there are trivial. 
For initial lymphatic segments the composition of lymph is generally considered to be similar to the interstitial fluid of the organs they drain. That also means that representing lymph with one compartment won't help much. For example IgG concentrations in hepatic lymph nodes are much higher than in e.g.  popliteal lymph nodes draining skin - reflecting the higher interstitial IgG concentration in liver.   
Lymph composition can generally change along the lymphatic tree due to a) mixing of lymph along the tree from different organs, b) fluid exchange across vessel walls and c) some exchange with blood plasma in lymph nodes. 
In what are you specifically interested? Using interstitial concentrations won't help?       

Regarding the steps to implement TMDD:

The rough steps are
1. Set up a PBPK model in PK-Sim including the binding reaction to the target
2. Export this model to MoBi
3. Add a reaction representing  internalization and degradation of the drug-target-complex in MoBi
4. Since target is degraded by this reaction: Also add a reaction representing target turnover (target synthesis and degradation) in MoBi 

These steps are explained in more detail in the following tutorial:
https://github.com/Open-Systems-Pharmacology/ISOP-webinar---PK-Sim-MoBi-introduction

Hope that helps.

Best regards,
Christoph

**Comment by wangwei1619** [comment]
Dear Christoph.

Thanks for your detailed explanation. 

When I firtst built a model for a kind of peptide (a little more than 1 KDa), I tried to do it as a small molecule. But the simulation results were accurate only when peptide was modeled to enter into cells (Clearance is fixed to values from literature; log P is -10; unbound fraction is 100%). I think it is not true because its receptor is on the surface of cells. So my initial guess is some peptide distributed into lymphatic circulation. Do you think it is resonable? If not, I may find other way such as target-mediated binding.

Thanks for your reply.

Best regards,
Wang Wei
